matter_server/client/models/node.py

```diff
@@ -69,9 +84,12 @@ def get_cluster(self, cluster: type[_CLUSTER_T] | int) -> _CLUSTER_T | None:
 
         Return None if the Cluster is not present on the node.
         """
-        if isinstance(cluster, type):
-            return self.clusters[cluster.id]  # type: ignore[no-any-return]
-        return self.clusters[cluster]  # type: ignore[no-any-return]
+        try:
+            if isinstance(cluster, type):
+                return self.clusters[cluster.id]  # type: ignore[no-any-return]
+            return self.clusters[cluster]  # type: ignore[no-any-return]
```

Comment:
this shouldn't happen, as it's always a type ?

Reply:
mypy was complaining to me but could be old mess